bluechi.spec.in

```diff
@@ -214,6 +214,7 @@ will be used during integration tests when creating code coverage report.
 %files coverage
 %license LICENSE
 %{_datadir}/bluechi-coverage/*.gcno
+%/var/tmp/bluechi-coverage/
```

Comment:
Directories should be using `%dir` macro and you should be using `%{_localstatedir}` instead of `/var` path:

```
%dir %{_localstatedir}/tmp/bluechi-coverage/
```